doc/sphinx/contributing.rst

```diff
@@ -85,10 +85,10 @@ of the build directory: ``doc/doxygen/html/index.html``.
 A recent version of this guide can also be found on the homepage
 http://espressomd.org/html/dox/.
 
-.. _Users guide:
+.. _Building the Users guide:
```

Comment:
does this link work without the tick?

Reply:
It's not linked anywhere, but any label works, right?
Changed that because of the conflicting labels with the UG and DG headings.

Reply:
its just a project convention to name labels exactly like the heading